src/Elastic.Markdown/Myst/InlineParsers/DiagnosticLinkInlineParser.cs

```diff
@@ -44,10 +47,14 @@ public override bool Match(InlineProcessor processor, ref StringSlice slice)
 			return match;
 
 		var url = link.Url;
+		var uri = Uri.TryCreate(url, UriKind.Absolute, out var u) ? u : null;
```

Comment:
Another slight nit pick we should move this down so we don't always try to do this (e.g if the url is null or empty). But this whole method is ripe for refactoring, we can do that at a later time.

Reply:
https://github.com/elastic/docs-builder/pull/168/commits/15055c0d5a23815d128de22a0a05e6867c0d5aeb